cmd/fortifyExecuteScan_test.go

```diff
@@ -402,6 +416,20 @@ func TestDetermineArtifact(t *testing.T) {
 	})
 }
 
+func TestFailFortifyexecinPath(t *testing.T) {
+	t.Run("Testing if fortifyupdate in $PATH or not", func(t *testing.T) {
+		ff := fortifyMock{}
+		utils := newFortifyTestUtilsBundle()
+		influx := fortifyExecuteScanInflux{}
+		auditStatus := map[string]string{}
+		execInPath = failMockExecinPath
+		config := fortifyExecuteScanOptions{SpotCheckMinimum: 4, MustAuditIssueGroups: "Audit All, Corporate Security Requirements", SpotAuditIssueGroups: "Spot Checks of Each Category"}
+		_, err := runFortifyScan(config, &ff, utils, nil, &influx, auditStatus)
+		assert.EqualError(t, err, "ERROR , command not found: fortifyupdate. Please configure a supported docker image or install Fortify SCA on the system.")
```

Comment:
Does this test succeed? The oracle message does not match the one in the mock. 

Let's stop for today and try to make a clean PR tomorrow.